cmake/configure.cmake

```diff
@@ -67,6 +67,12 @@ else()
     include_directories(${CUDA_TOOLKIT_INCLUDE})
 endif(NOT WITH_GPU)
 
+if(WITH_MKLDNN)
+    add_definitions(-DPADDLE_USE_MKLDNN)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
```

Comment:
Okay, I thought `OpenMP` would be necessary on most OS.
Two option:
1. Fix the two variable `-fopenmp`, just as mkldnn code did.
2, find openmp, give a warnning if can not found when use mkldnn.
By the way can you at least found `libgomp.so` in your system?